docs/docs/gatsby-core-philosophy.md

```diff
@@ -1,8 +1,166 @@
+
+We believe that in 5 or 10 years, we'll look back at many of these blocks like we look back at machine code or assembly language today; low-level languages that are great compile targets for higher-level languages that are easier to write in.
+
+Using abstractions like React components, `gatsby-image`, and `gatsby-link`, we've begun to craft this higher-level language. But we're just getting started. Gatsby is a playground for discovering new building blocks for the web.
+
+To [quote Alan Kay](https://www.youtube.com/watch?v=NdSD07U5uBs):
+
+> “You get simplicity by finding slightly more sophisticated building blocks“.
+
+[Read more here](/blog/2017-10-16-making-website-building-fun/#find-the-right-building-blocks).
+
+### Create a cohesive "content mesh system"
+
+Over the last few years, teams have begun moving from monolithic, integrated CMS setups to a modular "content mesh" pulling content and functionality from multiple sources and APIs. Their key challenge: pragmatically integrating all these pieces together.
+
+Gatsby's goal is to build a "content mesh system" that knits modular, best-of-breed tools in content modeling, authentication, search, analytics, payments, UI frameworks, and so on -- into a unified, high-quality whole.
+
+Read more on a [journey to the content mesh](/blog/2018-10-04-journey-to-the-content-mesh).
+
+### Make building websites fun
+
+Gatsby's vision is to make website development fun by making it simple.
+
+Fred Brooks wrote in _The Mythical Man-Month_:
+
+> The programmer, like the poet, works only slightly removed from pure thought-stuff. [They] build [their] castles in the air, from air, creating by exertion of the imagination. Few media of creation are so flexible, so easy to polish and rework, so readily capable of realizing grand conceptual structures…
+
+Technology is incredibly fun when we, like the wizard of fantasy, can type an incantation on our computer and a new creation comes to life.
+
+When this incantation is simple and takes seconds, we can easily get lost for hours in the rush of creation trying one thing after another improvising our way to an eventual design.
+
+Developers, designers, demand-gen marketers, copywriters -- every stakeholder in a website project _can_ feel this this -- with proper tooling.
+
+As Bret Victor suggested in _Inventing on Principle_, each creator needs "an immediate connection to what they're making.”
+
+These instant feedback loops are often felt near the beginning of projects. But they can quickly be drowned by project complexity.
+
+Developers can spend hours waiting for deploys, tracking down phantom "tooling" bugs, and so on.
+
+We believe that with the right building blocks, and an integrated "content mesh system", website projects should continue to feel "fun" no matter how big they get. We believe that every member of a website team deserves the ability to easily see how their work fits into the whole.
```

Comment:
Quoted "fun" makes me think of air-quotes. This reminded me of [this blog about unintentionally hilarious uses of quotes](http://www.unnecessaryquotes.com/2015/09/authentic.html).

Also dropped the quotes on the content mesh — we should probably start treating that as a real term and just link to the post for reference.

```suggestion
We believe that with the right building blocks, and an integrated content mesh system, website projects should continue to feel fun no matter how big they get. We believe that every member of a website team deserves the ability to easily see how their work fits into the whole.
```

Reply:
That makes sense. This is the first use of _content mesh system_, but let's try using it without air quotes and see if it sticks :)